src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapBad.java

```diff
@@ -1,11 +1,13 @@
 package com.google.checkstyle.test. //warn
               chapter3filestructure.rule332nolinewrap;
 
-import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
+import static java.math. //warn
+        BigInteger.ZERO;
 
-import javax.accessibility. //warn
-    AccessibleAttributeSequence;
+import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater; //ok
+import javax.accessibility. //warn
+        AccessibleAttributeSequence;
```

Comment:
I'll make the changes as you suggested but is it ok to defy the import order as per google's java guidelines [here](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing) even if it is "Bad" example? I reordered the imports as given in the link. Still no worries I'll make the changes as you said